docs/Technical-Documentation/clean-and-reparse.md

```diff
@@ -201,22 +201,22 @@ space:          tanf-dev
 $ /tmp/lifecycle/shell
 ```
 
-### 4. Display Help for Re-parse Command
+### 4. Display Help for Reparse Command
 ```bash
 $ python manage.py clean_and_reparse -h
 
 usage: manage.py clean_and_parse [-h] [-q {Q1,Q2,Q3,Q4}] [-y FISCAL_YEAR] [-a] [-n] [-d] [--configuration CONFIGURATION] [--version] [-v {0,1,2,3}] [--settings SETTINGS] [--pythonpath PYTHONPATH] [--traceback] [--no-color] [--force-color] [--skip-checks]
 
-Delete and re-parse a set of datafiles. All re-parsed data will be moved into a new set of Elastic indexes.
+Delete and reparse a set of datafiles. All reparsed data will be moved into a new set of Elastic indexes.
 
 options:
   -h, --help            show this help message and exit
   -q {Q1,Q2,Q3,Q4}, --fiscal_quarter {Q1,Q2,Q3,Q4}
-                        Re-parse all files in the fiscal quarter, e.g. Q1.
+                        Reparse all files in the fiscal quarter, e.g. Q1.
   -y FISCAL_YEAR, --fiscal_year FISCAL_YEAR
-                        Re-parse all files in the fiscal year, e.g. 2021.
-  -a, --all             Clean and re-parse all datafiles. If selected, fiscal_year/quarter aren't necessary.
-  -n, --new_indices     Move re-parsed data to new Elastic indices.
+                        Reparse all files in the fiscal year, e.g. 2021.
+  -a, --all             Clean and reparse all datafiles. If selected, fiscal_year/quarter aren't necessary.
+  -n, --new_indices     Move reparsed data to new Elastic indices.
```

Comment:
@andrew-jameson @jtimpe @elipe17 @raftmsohani can this portion of the documentation be update to remove the `-n` and `-d` elements? 

```shell=
vcap@...:~$ python manage.py clean_and_reparse -y 2022 -q Q1 -d -n
usage: manage.py clean_and_reparse [-h] [-q {Q1,Q2,Q3,Q4}]
                                   [-y FISCAL_YEAR] [-a]
                                   [--configuration CONFIGURATION]
                                   [--version] [-v {0,1,2,3}]
                                   [--settings SETTINGS]
                                   [--pythonpath PYTHONPATH]
                                   [--traceback] [--no-color]
                                   [--force-color] [--skip-checks]
manage.py clean_and_reparse: error: unrecognized arguments: -d -n

```